src/linear_algebra/basic.lean

```diff
@@ -1538,18 +1530,12 @@ def comap_mkq.rel_iso :
 
 /-- The ordering on submodules of the quotient of `M` by `p` embeds into the ordering on submodules
 of `M`. -/
-def comap_mkq.le_rel_embedding :
-  ((≤) : submodule R p.quotient → submodule R p.quotient → Prop) ↪r ((≤) : submodule R M → submodule R M → Prop) :=
+def comap_mkq.order_embedding :
+  submodule R p.quotient ↪o submodule R M :=
 (rel_iso.to_rel_embedding $ comap_mkq.rel_iso p).trans (subtype.rel_embedding _ _)
 
 @[simp] lemma comap_mkq_embedding_eq (p' : submodule R p.quotient) :
-  comap_mkq.le_rel_embedding p p' = comap p.mkq p' := rfl
-
-/-- The ordering on submodules of the quotient of `M` by `p` embeds into the ordering on submodules
-of `M`. -/
-def comap_mkq.lt_rel_embedding :
```

Comment:
I saw that you deleted `lt_rel_embedding` in a few places. Is that because you changed all the relevant lemmas to take a hypothesis of type `order_embedding` instead?

Reply:
Yes. Essentially I merged the `le_rel_embedding`s and the `lt_rel_embedding`s into one order embedding, and then when it's needed, I use `order_embedding.lt_embedding` if necessary.

Reply:
That seems worth noting in the PR comment (which becomes the final commit message).

Reply:
Done